app/src/main/kotlin/GameConfig.kt

```diff
@@ -0,0 +1,28 @@
+import org.hexworks.zircon.api.CP437TilesetResources
+import org.hexworks.zircon.api.ColorThemes
+import org.hexworks.zircon.api.data.Size
+import org.hexworks.zircon.api.data.Size3D
+
+/*
+Here are implemented all base
+constants for game setup
+ */
+
+object GameConfig {
```

Comment:
```suggestion
/**
  * Here are implemented all base
  * constants for game setup
  */
object GameConfig {
```
И ниже, надо комментарии оформить в формат KDoc